cloudtests.py

```diff
@@ -62,16 +62,34 @@ def test_net_name(self):
 class TestJournalWritten(unittest.TestCase):
 
     def test_journal_written(self):
```

Comment:
I believe we decided that we want this test to run on first boot as well as after reboot. Should we create a 2nd function called `test_journal_written_after_reboot()` that just calls `test_journal_written()`. This would make it easier to know which one failed when viewing the testing output.